README.md

```diff
@@ -80,4 +80,88 @@ instance applyV :: (Semigroup err) => Apply (V err)
 
 ``` purescript
 instance applicativeV :: (Semigroup err) => Applicative (V err)
+```
+
+
+
+## Module Data.Validation.Semiring
+
+
+This module defines an applicative functor and alt instance for 
```

Comment:
Could you change these to `Applicative` and `Alt` (capitalized, code blocks) please?

Also "instance_s_".